### PR TITLE
OAUTH2-157 Each feature should render as a block element, just like grant types

### PR DESCRIPTION
**Pull request by csierra**

### CODE REVIEW

**Comment by brianchandotcom** [comment]
@csierra 

Please resend..

```
oauth2.scope.DELETE=Create/Update/Delete data on your behalf
oauth2.scope.GET=Read data you can access
oauth2.scope.HEAD=Read data you can access
oauth2.scope.OPTIONS=Read data you can access
oauth2.scope.POST=Create/Update/Delete data on your behalf
oauth2.scope.PUT=Create/Update/Delete data on your behalf
oauth2.scope.everything=Unrestricted access on your behalf
oauth2.scope.everything.read=Read data you can access
oauth2.scope.everything.write=Create/update/delete data on your behalf

```

These need to be titles or sentences.

**Comment by stian-sigvartsen** [comment]
Hi @brianchandotcom . These are shown as tags and checkbox labels on the UI, and sometime concatenated together with "+" characters. So we can't end them in a full stop, which I understand is our normal convention. Also, if we make them too wordy then it convolutes the UI.  @sez11a - maybe can you suggest alternative wording?

screen shot 2018-06-14 at 14 09 38
